hash/ibm5170_cdrom.xml

```diff
@@ -197,7 +233,36 @@
 			<feature name="disk_label" value="CD0141" />
 			<diskarea name="cdrom">
 				<disk name="pompei" sha1="964c914b8580a21b2f1a54e9329336ead243598b" />
-			</diskarea>			
+			</diskarea>
+		</part>
+	</software>
+  
+	<software name="prototyp">
```

Comment:
prototyp was already added in one of your previous pull requests so this is a duplicate and causes a ```mame -valid``` failure